.travis.yml

```diff
@@ -95,13 +89,27 @@ install:
 
   # Update the package list and install the Cloud SDK
   - sudo apt-get -qq update
-  - sudo apt-get install google-cloud-sdk -y
+  - sudo apt-get -qq install google-cloud-sdk -y
+
+  # Installing Azure command-line client here as used both in cleaning and cron_job scripts
+  - echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+  - sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+  - sudo apt-get -qq install apt-transport-https -y
+  - sudo apt-get -qq update && sudo apt-get -qq install azure-cli -y
 
 before_script:
-  # Get current_version, variable common to all three platforms
-  - source ./bin/get_current_version.sh
+  # Check whether this build was triggered by our cron job for security-updates
+  - echo "TRAVIS_EVENT_TYPE is:$TRAVIS_EVENT_TYPE"
+  - >
+    if [ "$TRAVIS_EVENT_TYPE" = 'cron' ]; then
+        echo "Travis built triggered via configured cron job"
+        source bin/cron_script.sh
+    else
+        # Get current_version, variable common to all three platforms
+        source ./bin/get_current_version.sh
+    fi
   - IMAGE_NAME="$IMAGE_NAME""$CURRENT_VERSION" && export IMAGE_NAME
-  - echo "Image Name is:" && echo $IMAGE_NAME
+  - echo "Image Name is:$IMAGE_NAME"
```

Comment:
@carmat88 please add a space after the `:`

Reply:
I have not found a way to introduce spaces before a global variable while executing `echo` in a yml file. That is why at first I was using two echo in order to create a sort of different way of printing out those values. Even single quotes do not help.